easybuild/easyconfigs/d/darshan-runtime/darshan-runtime-3.4.4-gompi-2023a-HDF5-1.14.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'darshan-runtime'
+version = '3.4.4'
+versionsuffix = '-HDF5-1.14.0'
+
+homepage = 'https://www.mcs.anl.gov/research/projects/darshan/'
+description = """Darshan is a scalable HPC I/O characterization tool. It is
+designed to capture an accurate picture of application I/O behavior, including
+properties such as patterns of access within files, with minimum overhead.
+The name is taken from a Sanskrit word for 'sight' or 'vision'.
+darshan-runtime instruments (MPI) applications."""
+
+toolchain = {'name': 'gompi', 'version': '2023a'}
+
+source_urls = ['https://ftp.mcs.anl.gov/pub/darshan/releases']
+sources = ['darshan-%(version)s.tar.gz']
+checksums = ['d9c9df5aca94dc5ca3d56fd763bec2f74771d35126d61cb897373d2166ccd867']
+
+start_dir = 'darshan-runtime'
+
```

Comment:
also needs a builddep on ('Autotools', '20220317'), here as prepare.sh runs autoreconf.

Reply:
Done in e24dca74bf23a2496eaeeea3f4a6361137286759